src/crystal/system/win32/fiber.cr

```diff
@@ -1,19 +1,47 @@
 require "c/memoryapi"
+require "c/sysinfoapi"
 require "c/winnt"
 
 module Crystal::System::Fiber
+  RESERVED_STACK_SIZE = LibC::DWORD.new(0x10000)
+
+  private class_getter page_size : LibC::DWORD do
+    LibC.GetNativeSystemInfo(out system_info)
+    system_info.dwPageSize
+  end
```

Comment:
Is there a reason for caching only `page_size`, not `page_size + RESERVED_STACK_SIZE`? The difference is probably negligible, just wondering...

Reply:
Also, I'm not sure if lazy evaluation (i.e. `getter` with block) is really the best choice here. We should expect this value to be used in practically every Crystal program. So fetching/calculating it eagerly might reduce some performance overhead.